integration-test/pom.xml

```diff
@@ -89,7 +89,12 @@
         <dependency>
```

Comment:
I don't think that updating the docker-thingy is relevant to the Spring bump :/

Reply:
You can rest assured that it very much is.

https://github.com/spotify/docker-client/issues/1030